using-mobile-extensions/adobe-analytics/analytics-api-reference.md

```diff
@@ -82,7 +99,9 @@ Force delete, without sending to Analytics, all hits being stored or batched on
 {% tab title="Android" %}
 ### clearQueue
 
-**Warning:** Use caution when manually clearing the queue. This process cannot be reversed.
+{% hint style="warning" %}
+Use caution when manually clearing the queue. This process cannot be reversed.
```

Comment:
I think this is meant to be: `... This operation cannot be reverted.`
Let's update it everywhere.